tutor/src/components/tours/custom/common.jsx

```diff
@@ -1,6 +1,7 @@
 import React        from 'react';
 
 import classnames   from 'classnames';
+import forEach      from 'lodash/forEach';
 
```

Comment:
FYI we can do:  `import { forEach } from 'lodash'`  `babel-plugin-lodash` will translate that to be the same.  Is a bit cleaner when we have multiple imports.